modules/ROOT/pages/administration/user-management.adoc

```diff
@@ -0,0 +1,154 @@
+
+== Roles of NOM users
+
+[%noheader]
+|===
+||
+
+|NOM Admin
+|Has access to all features of NOM for all DBMSs  and can administer NOM users, including creating other users that have the role *"NOM Admin"*.
+
+|Administrator
+|Has access to features of NOM as configured by NOM Admin.
+Can not administer NOM users.
+|===
+
+[NOTE]
+====
+The default NOM user `admin` has the role *NOM Admin* and can therefore be used to create new NOM users (including other *Administrators*) and configure their access to NOM features.
+====
+
+== Scope of NOM users
+
+NOM users can only be used to access the functionality of the NOM server and are not related to other systems (Neo4j databases (persistence or monitored ones), operating system, etc.).
+
+== NOM user administration tasks
+
+The NOM user management is performed in the *Administrators* section of the NOM settings page.
+
+image::nom-administrators-section.png[width=800]
+
+=== Creating a user
+
+Click on *"Create user"* button to open the sidebar containing the tabs *"User info"* and *"DBMS access control"*.
+Fill in user details including specifying if role *"NOM Admin"* should be assigned to them.
+
+[IMPORTANT]
+====
+New users will be automatically redirected to the password change page after the login.
+====
+
+image::nom-administrators-create-user.png[width=800]
+
+If the new user will have the role *NOM Admin*, no further details must be specified.
+Use the *"Create"* button to confirm user creation.
+
+Access levels for global features can be set on the *"User info"* tab.
+Access to the *"Agents"* page of the NOM settings area is currently only one global feature that requires explicit permission.
+Applicable access levels are: *"No access"*, *"view"*, *"edit"*.
+
+==== Assigning access levels on DBMS specific features
+
+Assigning access levels on DBMS specific features is performed in the *"DBMS access control"* tab.
+
+For users having the role *Administrator* (checkbox *NOM Admin* on the *"User info"* tab is *OFF*), access to every DBMS specific feature can be given explicitly or implicitly.
+Explicit permissions are be assigned directly to DBMSs.
+Implicit permissions are assigned to DBMS glob patterns.
+
+image::nom-administrators-dbms-access-control.png[width=800]
+
+See the following table for permission description.
+
+|===
+|DBMS-specific feature |Access levels |Description
+
+|*Status*
+|View, edit
+|Access to the *"Status"* page for assigned DBMS(s).
+
+|*Security*
+|No access, view, edit
+|Access to the *"Security panel"* page for assigned DBMS(s).
+
+|*Status*
+|No access, view
+|Access to the *"Logs"* page for assigned DBMS(s).
+
+|*Upgrades*
+|No access, View, edit
+|Access to the *"Upgrade"* page for assigned DBMS(s).
+|===
+
+[IMPORTANT]
+====
+If the DBMS is renamed or migrated after being assigned explicit permissions, the permissions remain unchanged.
+====
+
+==== Duplicating permissions of another user
```

Comment:
I think it makes sense to switch this section with the next one `Working with DBMS glob patterns` because that would my first question from the previous section would be `How do globs work?`

Reply:
@eve-bright Eve, please see 24a8ab3. Directly swapping did not work for me so I had to move parts around :-)